### ISSUE TITLE
The documentation statement explaining the shared option is contradicting with itself

### ISSUE THREAD

**Issue by omnir95** [Xof/django-pglocks#29]
The following paragraph in the documentation is contradicting with itself:

> shared (default False) – If True, a shared lock is taken. **Any number of sessions can hold a shared lock**; if another session attempts to take an exclusive lock, it will wait until all shared locks are released; **if a session is holding a shared lock, it will block attempts to take a shared lock**. If False (the default), an exclusive lock is taken.

The bold parts are the contradicting parts. How can any number of sessions hold a shared lock while holding a shared lock will block the attempts to take a shared lock?

**Comment by kamilturek** [comment]
@omnir95 Yeah, I also spotted this today. It's apparently wrong. I've prepared a PR to amend that.
